composer.json

```diff
@@ -16,8 +16,10 @@
     ],
     "require": {
         "php": "^7.4|^8.0",
+        "illuminate/contracts": "^8.0",
         "spatie/laravel-package-tools": "^1.1",
-        "illuminate/contracts": "^8.0"
+        "stripe/stripe-php": "^7.69",
```

Comment:
Should we pull in laravel/cashier instead of stripe directly?

Reply:
I think that would be a better approach, but we will need to refactor a decent amount of our code throughout the application. Let's get all the packages created and released first and then add that as a priority.